### PR TITLE
[12.0][MIG] account_move_reconcile_helper

### PR DESCRIPTION
**Pull request by Abranes**
Standard migration

### CODE REVIEW

**Comment by Abranes** [comment]
I don't know why runbot fail, [account_balance_line](https://github.com/OCA/account-financial-tools/tree/12.0/account_balance_line) is migrated to 12.0.

Can anyone help me?

**Comment by pedrobaeza** [comment]
You have to add `oca_dependencies.txt` file in root for adding extra OCA repositories for dependencies.

**Comment by Abranes** [comment]
Oh! Sorry, I miss that! Now it's done and :green_apple:!

cc @benwillig @Jortolsa @fuentes010 @ioans73 @Saphery @Willerr-mo

**Review by fuentes73 — APPROVED** [review]

File: account_move_reconcile_helper/views/account_move_line.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="account_move_line_tree_view">
```
Comment (tbaden) [inline account_move_reconcile_helper/views/account_move_line.xml]: aesthetic change: 
id before model

use original id

> In the case where an inherited view has a name which does not follow the guidelines set above, prefer naming the inherited view after the original over using a name which follows the guidelines. This eases looking up the original view and other inheritance if they all have the same name.

not approve relevant

File: account_move_reconcile_helper/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/account-reconcile',
+    'depends': [
+        'account_balance_line',
+    ],
+    'data': [
+        'views/account_move_line.xml',
+    ],
+    'demo': [
```
Comment (tbaden) [inline account_move_reconcile_helper/__manifest__.py]: if you dont use this tag, just delete it

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for your work 👍 
just a minor change (see below)
and please squash your commits.

**Review by benwillig — APPROVED** [review]
LGTM. Thanks for the migration

**Comment by Abranes** [comment]
@tbaden Squash and minor changes done!

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM!

**Review by Jortolsa-S73 — APPROVED** [review]

**Comment by Abranes** [comment]
@pedrobaeza Can you review this?

**Comment by pedrobaeza** [comment]
Merging blindly as it seems there are enough reviewers.

/ocabot merge

**Pull request closed without merging**